### PR TITLE
r.geomorphon more cleanups

### PR DESCRIPTION
**Pull request by infrastation**
These should be the last changes before the new feature I've been working on, please review.

### CODE REVIEW

File: raster/r.geomorphon/main.c
Comment (wenzeslaus) [inline raster/r.geomorphon/main.c:162]: I have no idea if the macros work with the translation function (`_(...)`), specifically with the extraction for strings for translations. Any ideas? @marisn @veroandreo @neteler

File: raster/r.geomorphon/r.geomorphon.html
Comment (wenzeslaus) [inline raster/r.geomorphon/r.geomorphon.html:98]: Only a minor issue: Does this fix also `document type does not allow element "H2" here; assuming missing "DD" start-tag`? I see it in the commit message, but I don't understand what is the change. Just trying to avoid missing it.

**Review by wenzeslaus — COMMENTED** [review]
I have some doubts about the macros for GUI section names, but I don't know, so I hope somebody else can chime in with some more insights. Otherwise, this looks good and it will ready to go after the translation issue is resolved.

File: raster/r.geomorphon/main.c
Comment (infrastation) [reply raster/r.geomorphon/main.c:162]: It would be fine to drop that specific commit to avoid confusion.

File: raster/r.geomorphon/r.geomorphon.html
Comment (infrastation) [reply raster/r.geomorphon/r.geomorphon.html:98]: Yes, with these changes the final HTML file passes the validation. The H2 element error was a consequence of the missing closing DL element.

Comment (infrastation) [reply raster/r.geomorphon/r.geomorphon.html:98]: ...or rather the closing tag of the DL element, for that matter.

File: raster/r.geomorphon/main.c
Comment (wenzeslaus) [reply raster/r.geomorphon/main.c:162]: Now, I'm wondering if the rest of the strings in that array, namely descriptions, are translated. Sending it to `_()` is not enough AFAIK. It really needs to be in `_()` to be picked for the translation files. With that, yes, drop it, so we can merge right away, but it would be good if somebody with translation experience checks this.

File: raster/r.geomorphon/r.geomorphon.html
Comment (wenzeslaus) [reply raster/r.geomorphon/r.geomorphon.html:98]: Thanks for clarifying it!

**Comment by infrastation** [comment]
Rebased on the most recent master branch and without the string macros this time. Likely going to fail "actions" for the same reason as before.

**Review by wenzeslaus — APPROVED** [review]

**Comment by wenzeslaus** [comment]
Perfect. Merged.

The Actions are currently down, so everything is failing right now. I already have a PR for it. Waiting for at least one review before merging that.

**Comment by infrastation** [comment]
Thank you.

File: raster/r.geomorphon/main.c
Comment (neteler) [reply raster/r.geomorphon/main.c]: A gettext macro on a variable AFAIK won't work as it will try to literally extract the variable name (but not expand it).

Comment (wenzeslaus) [reply raster/r.geomorphon/main.c]: Thanks @neteler . Actually, even in the current code, the documentation might not be translated, so that's something to keep an eye on.

**Pull request merged**
